### ISSUE TITLE
Personal custom character not showing up in menu

### ISSUE THREAD

**Issue by Samuel-Harper** [devyndamonster/TakeAndHoldTweaker#45]
I decided this update to work on my own simple character for fun, however despite having no json syntax errors and basically following the existing custom characters as a basis along with copying the manifest with the names swapped out from Timmy, my character doesn't show up in the game while the base 3 custom characters do.

I turned on logging to see what the issue could be and got hit with this in the logs before the other custom characters are loaded.
`[Warning:      Deli] Failed to create mod from archive: BJBlazkomeat.zip`

I've also tried different values in
`  "CharacterID": 2,
  "CharacterGroup": 2,`

My manifest is also just:
```
{
  "guid": "h3vr.tnhtweaker.bjblazkomeat",
  "version": "0.1.1.0",
  "dependencies": {
      "h3vr.tnhtweaker.deli": "0.1.4.0"
  },
  "name": "BJ Blazkomeat",
  "authors": [ 
    "Sam Harper" 
  ],
  "runtime": {
    "BJBlazkomeat/": "Character"
  }
}
```
It's probably also worth mentioning I am using custom images for the store and thumbnail but the store ones are based on the template found in the custom characters, and the thumbnail is the exact same size, format, and name.

If you need anything else to diagnose what's happening let me know.

**Comment by devyndamonster** [comment]
Could you post the log file found in BepInEx/OutputLog.log. That will give me a better idea of where the issue is. Also, if you could send screenshots of your mods folder, and inside your character mod.

**Comment by Samuel-Harper** [comment]
The output log can be found here: https://pastebin.com/LGE8ALpf
Here is my /mods folder: https://i.imgur.com/heCvPUM.png
And the inside of the uncompressed character folder is: https://i.imgur.com/dj07uXZ.png and https://i.imgur.com/aNTb5rR.png

Also worth mentioning that I recently got the [John Wurst character](https://bonetome.com/h3vr/characters/195/) and that shows up fine in the menus.

**Comment by Samuel-Harper** [comment]
I solved the issue, and I'm really embarrassed to admit the fault.

So I had a folder which contained the files I worked on, and I would right click and compress that folder, well maybe because I usually develop on Linux but I didn't realize this puts the folder itself in a zip instead of the contents of the folder, so the manifest became buried in a folder in a zip and unreadable to the mod loader.

After realizing this and fixing it the mod loaded fine and in the list of playable characters.

Thanks for helping though and hopefully someone can use my blunder to realize their own mystery error.
